src/helpers.js

```diff
@@ -65,6 +66,34 @@ export default {
       allureReporter.addAttachment('Execution video', videoPath, 'video/mp4');
     }
 
+    const frames = glob.sync(`${this.recordingPath}/*.png`);
+
+    if (frames.length) {
+      const frameRegex = /^.*\/(\d{4})\.png$/;
+      const frameNumbers = frames.map((path) => +path.replace(frameRegex, '$1'));
+      const pad = (frameNumber) => frameNumber.toString().padStart(4, '0');
+      const insertMissing = (sourceFrame, targetFrame) => {
+        const src = `${this.recordingPath}/${pad(sourceFrame)}.png`;
+        const dest = `${this.recordingPath}/${pad(targetFrame)}.png`;
+        const options = {overwrite: false};
+        fs.copySync(src, dest, options);
```

Comment:
same here, let's add `import fsp from 'node:fs/promises'` and do
```suggestion
        await fsp.copy(src, dest, options);
```